modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/api/lifecycle/AccessibilityOfRetireAPITestCase.java

```diff
@@ -0,0 +1,139 @@
+
+        //Invoke  old version
+        HttpResponse oldVersionInvokeResponse =
+                HttpRequestUtil.doGet(API_BASE_URL + API1_CONTEXT + "/" + API_VERSION1 + API1_END_POINT_METHOD,
+                        requestHeaders);
+        assertEquals(oldVersionInvokeResponse.getResponseCode(), HTTP_RESPONSE_CODE_OK,
+                "Response code mismatched when invoke api before Retire");
+        assertTrue(oldVersionInvokeResponse.getData().contains(API1_RESPONSE_DATA),
+                "Response data mismatched when invoke  API  before Retire" +
+                        " Response Data:" + oldVersionInvokeResponse.getData());
+
+    }
+
+
+    @Test(groups = {"wso2.am"}, description = "Change API lifecycle to Retired",
+            dependsOnMethods = "testInvokeAPIBeforeChangeAPILifecycleToRetired")
+    public void testChangeAPILifecycleToRetired() throws Exception {
+        //Block the API version 1.0.0
+        APILifeCycleStateRequest blockUpdateRequest =
+                new APILifeCycleStateRequest(API1_NAME, API1_PROVIDER_NAME, APILifeCycleState.RETIRED);
+        blockUpdateRequest.setVersion(API_VERSION1);
+        //Change API lifecycle  to Block
+        HttpResponse blockAPIActionResponse =
+                apiPublisherClientUser1.changeAPILifeCycleStatusTo(blockUpdateRequest);
+        assertEquals(blockAPIActionResponse.getResponseCode(), HTTP_RESPONSE_CODE_OK, "Response code mismatched");
+        assertTrue(verifyAPIStatusChange(blockAPIActionResponse, APILifeCycleState.PUBLISHED,
+                APILifeCycleState.RETIRED), "API status Change is invalid when retire an API :" +
+                getAPIIdentifierString(apiIdentifierAPI1Version1) +
+                " Response Code:" + blockAPIActionResponse.getData());
+
+    }
+
+
+    @Test(groups = {"wso2.am"}, description = "Test the availability of retired API in the store",
+            dependsOnMethods = "testChangeAPILifecycleToRetired")
+    public void testAvailabilityOfRetiredAPIInStore() throws Exception {
+        //  Verify the API in API Store : API should not be available in the store.
+        List<APIIdentifier> apiStoreAPIIdentifierList = APIMgtTestUtil.getAPIIdentifierListFromHttpResponse(
+                apiStoreClientUser1.getAPI(API1_NAME));
+        assertEquals(APIMgtTestUtil.isAPIAvailable(apiIdentifierAPI1Version1, apiStoreAPIIdentifierList), false,
+                "Api is  visible in API Store after retire." + getAPIIdentifierString(apiIdentifierAPI1Version1));
+
+    }
+
+
+    @Test(groups = {"wso2.am"}, description = "Test the invocation of the API after retire",
+            dependsOnMethods = "testAvailabilityOfRetiredAPIInStore")
+    public void testInvokeAPIAfterChangeAPILifecycleToRetired() throws Exception {
+
+        //Invoke  old version
+        HttpResponse oldVersionInvokeResponse =
+                HttpRequestUtil.doGet(API_BASE_URL + API1_CONTEXT + "/" + API_VERSION1 + API1_END_POINT_METHOD,
```

Comment:
API1_CONTEXT  and  API_VERSION1 variable names are inconsistent. Please use more consistent names for these variables.